app/helpers/catalog_helper.rb

```diff
@@ -12,4 +12,32 @@ def render_document_heading(*args)
     val = Array.wrap(presenter(document).heading).map { |v| h(v) }.join("<br />")
     content_tag(tag, val, { itemprop: "name", dir: val.to_s.dir }, false)
   end
+
+  def can_edit?
```

Comment:
What's the benefit of having these in helpers rather than just having `can?(:update, resource)` in the view?

Reply:
Slightly more complex cases (e. g. `can?(:edit, resource) && can?(:create, EphemeraFolder)`) felt to cluttered for the View layer, but I can shift these into the View template